include/prereq.mk

```diff
@@ -103,7 +103,7 @@ define SetupHostCommand
 	           $(call QuoteHostCommand,$(11)) $(call QuoteHostCommand,$(12)); do \
 		if [ -n "$$$$$$$$cmd" ]; then \
 			bin="$$$$$$$$(PATH="$(subst $(space),:,$(filter-out $(STAGING_DIR_HOST)/%,$(subst :,$(space),$(PATH))))" \
-				which "$$$$$$$${cmd%% *}")"; \
+				command -v "$$$$$$$${cmd%% *}")"; \
```

Comment:
Is this correct?

Reply:
At a quick glance that looks correct. PATH seems to be set above.

Reply:
on the original pr -pv was used wonder why?

Reply:
most likely a mistake when converting to which.